common/src/main/scala/explore/AppMain.scala

```diff
@@ -115,6 +118,17 @@ trait AppMain extends IOApp {
       }
     )
 
+    // Creates a "profile" for user preferences.
+    def createUserPrefs(vault: Option[UserVault], ctx: AppContext[IO]): IO[Unit] = {
+      implicit val db = ctx.clients.preferencesDB
```

Comment:
If you pass `AppContext[IO]` implicitly and `import explore.impicits._` then you don't need this line (the  `db` is implicitly derived from `ctx`).